src/py42/clients/alerts.py

```diff
@@ -136,3 +136,32 @@ def get_aggregate_data(self, alert_id):
             :class:`py42.response.Py42Response`
         """
         return self._alert_service.get_aggregate_data(alert_id)
+
+    def get_all_alert_details(self, query, ascending=True):
```

Comment:
Should this maybe be called `search_all_alert_details` or `search_alert_details` instead? As written might give the impression that we are retrieving **all** alert details in the database

Reply:
Yeah, I could go either way. I was thinking of this more as a corollary to `get_details()` but without the 100 alert limitation and also not having know the ids themselves. I also thought about `iter_all_alert_details` since we're iterating over the alerts individually opposed to a "page" that all the other methods return.

Reply:
Took a look at some of the other py42 clients and looks like we use `get_all` to mean "get all but we also support several filtering parameters". So I think in that context this usage is accurate and consistent.